pystan/tests/test_covexpquad.py

```diff
@@ -6,19 +6,19 @@ class TestCovExpQuad(unittest.TestCase):
     """
     Unit test added to address issue discussed here: https://github.com/stan-dev/pystan/issues/271
     """
-    
-    covexpquad = """
-        data {
-        real rx1[5];
-    }
-    model {
-        matrix[5,5] a;
+    @classmethod
+    def setUpClass(cls):
+      covexpquad = """
+          data {
+          real rx1[5];
+      }
+      model {
+          matrix[5,5] a;
 
-        a = cov_exp_quad(rx1, 1, 1);
-    }
-        """
-
-    model = StanModel(model_code=covexpquad)
+          a = cov_exp_quad(rx1, 1, 1);
+      }
+          """
+      self.model = StanModel(model_code=covexpquad)
```

Comment:
``self`` needs to be ``cls``. Otherwise this is a good change, thanks!

Reply:
Thanks for catching this!